app/Console/Commands/AchievementsCron.php

```diff
@@ -513,4 +519,26 @@ private function bigKid()
         }
         return $selected;
     }
+
+    private function percentageActivities($percentage) {
+        $selected = [];
+            if (Carbon::now()->day == 1) {
+                $users = User::all();
+                foreach ($users as $user) {
+                    $participated = ActivityParticipation::where('user_id', $user->id)->pluck('activity_id');
+                    $activities = Activity::whereIn('id', $participated)->pluck('event_id');
+                    $CountEventsParticipated = Event::whereIn('id', $activities)->where('end','>',Carbon::now()->subMonth()->valueOf())->where('end','<',Carbon::now()->valueOf())->where('secret','=', '0')->count();
+
+                    $possibleActivities = Event::where('end','>',Carbon::now()->subMonth()->valueOf())->where('end','<',Carbon::now()->valueOf())->where('secret','=', '0')->has('activity')->count();
+
+                    if (floor($CountEventsParticipated / $possibleActivities * 100) >= $percentage) {
+                        $selected[] = $user;
+                    }
+                }
+            }
+            else {
+                $this->info('Its not the first of the month! Cancelling '.$percentage.' of Activities this month!');
```

Comment:
I don't quite understand this message. What does "Canceling `percentage` of activities this month!" mean?

Reply:
I have updated the cancellation message to be more clear.